docs/Mounting-and-Transparent-Naming.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: global
+title: Mounting API and Transparent Naming
+nickname: Mounts and Transparent Naming
+group: Features
+priority: 4
+---
+
+* Table of Contents
+{:toc}
+
+Tachyon enables effective data management across different storage systems through its use of
+transparent naming and mounting API.
+
+## Transparent Naming
+
+Transparent naming maintains an identity between the Tachyon namespace and the underlying storage
+system namespace.
+
+![transparent](./img/screenshot_transparent.png)
+
+When a user creates objects in the Tachyon namespace, they can choose whether these objects should
+be persisted in the underlying storage system. For objects that are persisted, Tachyon preserve the
+object paths, relative to the underlying storage system directory in which Tachyon objects are
+stored. For instance, if a user creates a top-level directory `Users` with subdirectories `Alice`
+and `Bob`, the directory structure and naming is preserved in the underlying storage system (e.g.
+HDFS or S3). Similarly, when a user renames or deletes a persisted object in the Tachyon namespace,
+it is renamed or deleted in the underlying storage system.
+
+Furthermore, Tachyon transparently discovers content present in the underlying storage system which
+was not created through Tachyon. For instance, if the underlying storage system contains a directory
+`Data` with files `Reports` and `Sales`, all of which were not created through Tachyon, their
+metadata will be loaded into Tachyon the first time they are accessed (e.g. when the user asks to
```

Comment:
Please specify whether the real data will be cached into Tachyon the first time they are accessed? If not, how to accomplish that?